nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/v1/HostedHandlers.java

```diff
@@ -41,43 +38,30 @@
 public class HostedHandlers
     extends ComponentSupport
 {
-  private static final String V1_CONANS = "/v1/conans/";
+  private static final String HOSTED_ASSET_PATH_PREFIX = "conans";
 
   private static final String CLIENT_CHECKSUM = "X-Checksum-Sha1";
 
   public final Handler uploadUrl = context -> {
     State state = context.getAttributes().require(TokenMatcher.State.class);
     AssetKind assetKind = context.getAttributes().require(AssetKind.class);
     ConanCoords coord = convertFromState(state);
-    String assetPath = getAssetPath(coord);
+    String assetPath = getHostedAssetPath(coord);
 
     return context.getRepository()
         .facet(ConanHostedFacet.class)
         .uploadDownloadUrl(assetPath, coord, context.getRequest().getPayload(), assetKind);
   };
 
-  public final Handler uploadManifest = context -> upload(context, "conanmanifest.txt");
-
-  public final Handler uploadConanFile = context -> upload(context, "conanfile.py");
-
-  public final Handler uploadConanInfo = context -> upload(context, "conaninfo.txt");
-
-  public final Handler uploadConanPackage = context -> upload(context, "conan_package.tgz");
-
-  public final Handler uploadConanSources = context -> upload(context, "conan_sources.tgz");
-
-  public final Handler uploadConanExport = context -> upload(context, "conan_export.tgz");
-
-  private Response upload(final Context context, final String filename) throws IOException {
-
+  public final Handler uploadContentHandler = context -> {
```

Comment:
I think it's better to put documentation above the variable. Look s a bit weird

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/91/commits/c39317f5cef05bd0906093aed821cbcb993407a9